modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/status/QuoteRequestedCommerceOrderStatusImpl.java

```diff
@@ -36,6 +42,7 @@
  * @author Brian I. Kim
  */
 @Component(
+	configurationPid = "com.liferay.commerce.configuration.CommerceOrderFieldsConfiguration",
```

Comment:
we are using the ConfigurationProvider class, I don't think we need to set the configuration pid